spec/spec_helper_acceptance.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'serverspec'
+require 'puppet_litmus'
+require 'spec_helper_acceptance_local' if File.file?(File.join(File.dirname(__FILE__), 'spec_helper_acceptance_local.rb'))
+include PuppetLitmus
+
+if ENV['TARGET_HOST'].nil? || ENV['TARGET_HOST'] == 'localhost'
+  puts 'Running tests against this machine !'
+  if Gem.win_platform?
+    set :backend, :cmd
+  else
+    set :backend, :exec
+  end
+else
```

Comment:
remove the else and replace with

  return
end

Then remove the else around the main body. You are done at this point if you are running locally. It cuts down on nesting and code coverage.